<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trigInputWindow</class>
 <widget class="QDialog" name="trigInputWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Triangle Solver</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="triangleBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>-30</y>
     <width>821</width>
     <height>571</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../assets/download.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="fullBaseBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>500</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>fullBase</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="thetaBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>370</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>theta</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="someLineBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>someLine</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="heightBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>height</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yoMamaBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>yoMama</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="notThetaBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>notTheta</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leftAngleBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>leftAngle</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rightBaseBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>rightBase</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rightAngleBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>rightAngle</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fullAngleBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>fullAngle</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leftBaseBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>leftBase</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>550</y>
     <width>510</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="isRad">
        <property name="text">
         <string>Radians</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="isDeg">
        <property name="text">
         <string>Degrees</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="solveSymbolic">
        <property name="text">
         <string>Solve Symbolically</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Recursive Solve Limit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="recursiveLimit">
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Solve For</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>leftAngle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>rightAngle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>fullAngle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>yoMama</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>someLine</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>height</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>theta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>notTheta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>leftBase</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>rightBase</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>fullBase</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Equation:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="equation">
        <property name="text">
         <string>Not enough information</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Answer:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="solution">
        <property name="text">
         <string>Not enough information</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>630</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calcButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>590</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>isRad</tabstop>
  <tabstop>isDeg</tabstop>
  <tabstop>solveSymbolic</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>leftAngleBox</tabstop>
  <tabstop>rightAngleBox</tabstop>
  <tabstop>fullAngleBox</tabstop>
  <tabstop>yoMamaBox</tabstop>
  <tabstop>someLineBox</tabstop>
  <tabstop>heightBox</tabstop>
  <tabstop>notThetaBox</tabstop>
  <tabstop>leftBaseBox</tabstop>
  <tabstop>fullBaseBox</tabstop>
  <tabstop>rightBaseBox</tabstop>
  <tabstop>thetaBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
